code/src/main/java/com/google/sps/data/BusinessNumber.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+
+// Keep track of information needed to contact a business
+public class BusinessNumber implements ContactStatus{
+
+  String nameOfBusiness;
+  long phoneNumber;
+  private long getTimeWhenCalledUnixTimeSeconds;
+  private boolean businessHasBeenCalled = false;
```

Comment:
Please use a constructor to initialize all fields.

Reply:
I am not sure what to initialize getTimeWhenCalledSeconds to

Reply:
perhaps that's a signal that it should be an optional long then! if it hasn't been called yet, we want to know that and have it cased differently than if it has been.

Reply:
okay